api_server.py

```
@@ -81,12 +81,23 @@ def get_crew_list() -> List[Dict]:
+        if table == "crew_availability":
+            curr = conn.execute(
+                "SELECT end_time FROM crew_availability WHERE crew_id = ? AND start_time <= ? AND end_time > ? LIMIT 1",
+                (entity_id, now, now),
+            ).fetchone()
+        elif table == "appliance_availability":
+            curr = conn.execute(
+                "SELECT end_time FROM appliance_availability WHERE appliance_id = ? AND start_time <= ? AND end_time > ? LIMIT 1",
+                (entity_id, now, now),
+            ).fetchone()
+        else:
+            logger.error(f"Invalid table name provided for availability check: {table}")
```

Ruff (0.15.6)

[warning] Logging statement uses f-string

(G004)